docs/troubleshooting/management/ts-management.asciidoc

```diff
@@ -78,3 +78,62 @@ image::images/transforms-start.png[Transforms page with Start option selected]
 . On the confirmation message that displays, click *Start* to restart the transform.
 . The transform’s status changes to `started`. If it doesn't change, refresh the page.
 ====
+
+[discrete]
+[[ts-agent-connection]]
+.{agent} and Endpoint connection issues
+[%collapsible]
+====
+
+After {agent} installs Endpoint, Endpoint connects to {agent} over a local relay connection to report its health status and receive policy updates and response action requests. If that connection cannot be established, the {elastic-defend} integration will cause {agent} to be in an `Unhealthy` status, and Endpoint won't operate properly.
+
+[float]
+==== Identify if the issue is happening
+
+You can identify if this issue is happening in the following ways:
+
+* Run {agent}'s status command:
+** `sudo /opt/Elastic/Agent/elastic-agent status` (Linux)
+** `sudo /Library/Elastic/Agent/elastic-agent status` (macOS)
+** `c:\Program Files\Elastic\Agent\elastic-agent.exe status` (Windows)
+
+If the status result for `endpoint-security` says that Endpoint has missed check-ins or `localhost:6788` cannot be bound to, it might indicate this problem is occurring.
+
+* If the problem starts happening right after installing Endpoint, check the value of `fleet.agent.id` in the following file:
+** `/opt/Elastic/Endpoint/elastic-endpoint.yaml` (Linux)
+** `/Library/Elastic/Endpoint/elastic-endpoint.yaml` (macOS)
+** `c:\Program Files\Elastic\Endpoint\elastic-endpoint.yaml` (Windows)
+
+If the value of `fleet.agent.id` is `00000000-0000-0000-0000-000000000000`, this indicates this problem is occurring.
```

Comment:
Small alignment tweak: this should get the ending paragraphs to align under the first-level bullets, instead of all the way to left-flush. (The problem isn't happening in the serverless version so no tweak needed there.)
```suggestion
* Run {agent}'s status command:
+
--
** `sudo /opt/Elastic/Agent/elastic-agent status` (Linux)
** `sudo /Library/Elastic/Agent/elastic-agent status` (macOS)
** `c:\Program Files\Elastic\Agent\elastic-agent.exe status` (Windows)
--
+
If the status result for `endpoint-security` says that Endpoint has missed check-ins or `localhost:6788` cannot be bound to, it might indicate this problem is occurring.

* If the problem starts happening right after installing Endpoint, check the value of `fleet.agent.id` in the following file:
+
--
** `/opt/Elastic/Endpoint/elastic-endpoint.yaml` (Linux)
** `/Library/Elastic/Endpoint/elastic-endpoint.yaml` (macOS)
** `c:\Program Files\Elastic\Endpoint\elastic-endpoint.yaml` (Windows)
--
+
If the value of `fleet.agent.id` is `00000000-0000-0000-0000-000000000000`, this indicates this problem is occurring.
```